<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputSelect</class>
 <widget class="QWidget" name="OutputSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OutputSelect</string>
  </property>
  <widget class="QLineEdit" name="outputLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="outputFileSelect">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output file:</string>
   </property>
  </widget>
  <widget class="QLabel" name="transcriptLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Transcript file:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="transcriptLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="transcriptFileSelect">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
